pkg/vc/rp/models.go

```diff
@@ -24,3 +30,43 @@ type DIDDocCredentialSubject struct {
 	ID     string `json:"id"`
 	DIDDoc string `json:"didDoc"`
 }
+
+// PresentationSubmissionPresentation is the PresentationSubmission VerifiablePresentation.
+// https://identity.foundation/presentation-exchange/#presentation-submission.
+type PresentationSubmissionPresentation struct {
+	Base                   *verifiable.Presentation               `json:"-"`
+	PresentationSubmission *presentationex.PresentationSubmission `json:"presentation_submission,omitempty"`
+}
+
+// CreditCardStatementCredential is the VerifiableCredential with a credit card statement.
+type CreditCardStatementCredential struct {
+	Base    *verifiable.Credential                `json:"-"`
+	Subject *CreditCardStatementCredentialSubject `json:"credentialSubject"`
+}
+
+// CreditCardStatementCredentialSubject is the custom credentialSubject of the CreditCardStatementCredential.
+type CreditCardStatementCredentialSubject struct {
+	ID        string              `json:"id"`
+	Statement CreditCardStatement `json:"stmt"`
+}
+
+// CreditCardStatement is a credit card statement.
+type CreditCardStatement struct {
```

Comment:
Why are we defining domain specific data model here ? How's this gonna scale ? eg, bankAccount, creditScore etc.  This doesn't look right.

Reply:
Suggestions are welcome.

Reply:
How can we validate the data contained withing the VCs? Or are we satisfied with the mere presence of a VC that the sender claims satisfies the presentation-definitions ?

Reply:
@rolsonquadras removed.

I've opened #120 as followup to our discussion this morning.